docs/workload-tutorial/README.md

```diff
@@ -253,3 +253,132 @@ To see what the updated source files should look like, refer to the files in
 directory
 [$TCF_HOME/docs/workload-tutorial/hello_world/stage_2/](hello_world/stage_2/).
 
+### <a name="phase3"></a>Phase 3: Worker-specific Code to execute IO opearations inside TEE
```

Comment:
change "opearations" to "operations"
change "inside TEE" to "inside the TEE"

Reply:
I would add an introductory paragraph.  Your commit text with minor revisions is a good introduction and I would place it here. Something like:
```
This tutorial defines step by step development of a workload with file IO handling capability.
For the first input from the client, the workload echoes user name along with an encryption key.
For the subsequent requests, the client submits a user name along with a key received in the previous step.
As a response, the client receives a username along with the umber of hello world workload invocations by that user.

```

Reply:
I have updated the introductory paragraph at the beginning of phase 3.